ipytree/tree.py

```diff
@@ -45,11 +46,12 @@ class Node(Widget):
 
     _id = Unicode(read_only=True).tag(sync=True)
 
-    def __init__(self, name="Node", nodes=[], **kwargs):
+    def __init__(self, name="Node", nodes=[], icon_image="", **kwargs):
```

Comment:
That's actually not needed to make this change. It will already be part of the `kwargs`. I actually added `name` and `nodes` because I thought it would be pretty common to always pass those two arguments. Maybe `icon_image` would be less common?

A benefit from explicitly adding the argument is that you get code completion on this argument in IPython. But I'm not fond of passing all the traits as explicit arguments, that would be too many of them.

Reply:
All right, let's remove this extra argument

Reply:
Updated